tests/device_tests/bitcoin/test_authorize_coinjoin.py

```diff
@@ -417,3 +417,30 @@ def test_multisession_authorization(client: Client):
         commitment_data=b"\x10www.example2.com" + (1).to_bytes(ROUND_ID_LEN, "big"),
         preauthorized=True,
     )
+
+
+@pytest.mark.setup_client(pin=PIN)
+def test_busy_state(client: Client):
```

Comment:
this should probably go to a separate `test_busy_state.py` outside `bitcoin` directory.

also please add an expiry test -- set busy for like 100 ms and check after sleeping that the device is no longer busy

Reply:
https://github.com/trezor/trezor-firmware/pull/2445/commits/fd02ad8a9b4eefce0bbe9fc27cb90568761256bd